lib/tasks/opened_closed_dates.rake

```diff
@@ -0,0 +1,18 @@
+desc "Find CMA Cases with opened dates that are before closed dates"
+task opened_before_closed_dates: :environment do
+  cma_cases = Services.publishing_api.get_content_items(
+    document_type: "cma_case",
+    per_page: 999999,
+  ).results
+
+  non_blank_opened_dates = cma_cases.select { |cma| !cma.details.metadata.opened_date.blank? }
+  non_blank_closed_dates = cma_cases.select { |cma| !cma.details.metadata.closed_date.blank? }
+
+  non_blank_opened_dates.each do |cma|
+    if non_blank_closed_dates.include? cma
+      unless cma.details.metadata.opened_date <= cma.details.metadata.closed_date
+        puts "Opened date is before closed date. Opened: #{cma.details.metadata.opened_date}, Closed: #{cma.details.metadata.closed_date}, content_id: #{cma.content_id}"
+      end
+    end
+  end
+end
```

Comment:
CMA is the only format that has `opened_date` and `closed_date`. As far as I can see the only other format with something similar is `Vehicle Recalls and Faults Alerts` which has `Build Start Date` and `Build End Date`. I would need to add a new validation method though due to the difference in field names... should I add it in?